app/app-maintenance.js

```diff
@@ -78,7 +64,7 @@ app.use(function (req, res, next) {
 
 // Development error handler.
 app.use(function (err, req, res, next) {
-  log.error({ error: err })
+  log.error(err)
```

Comment:
The log format is different so this used to log the entire object. May still need tweaks.